packages/docs/guide/essentials/passing-props.md

```diff
@@ -81,7 +81,7 @@ Try to keep the `props` function stateless, as it's only evaluated on route chan
 
 ## Via RouterView
 
-You can also pass any props directly via `<RouterView>`:
+You can also pass any props via the [`<RouterView>` slot](/guide/advanced/router-view-slot):
```

Comment:
Easier for translations:

```suggestion
You can also pass any props via the [`<RouterView>` slot](../advanced/router-view-slot):
```

Reply:
There were a few other new links that I've also updated to use relative paths.